src/interfaces/IWell.sol

```diff
@@ -16,6 +16,17 @@ struct Call {
  * @title IWell is the interface for the Well contract.
  */
 interface IWell {
+    /// @notice Returns the slippage of swapping `amountOut` of `fromToken` for `toToken`.
```

Comment:
Let's shoot for a more descriptive error message here.

This does not return slippage; this error is thrown when the `amountOut` doesn't pass the `minAmountOut` threshold.

```suggestion
    /// @notice Thrown when an operation would deliver fewer tokens than `minAmountOut`.
```